imports/plugins/core/address/register.js

```diff
@@ -1,33 +1,4 @@
 import Reaction from "/imports/plugins/core/core/server/Reaction";
-import { registerPluginHandler } from "./server/no-meteor/registration";
-import queries from "./server/no-meteor/queries";
-import resolvers from "./server/no-meteor/resolvers";
-import schemas from "./server/no-meteor/schemas";
+import register from "./server/no-meteor/register";
 
-/**
- * @file Address core plugin
- *
- * @namespace Address
- */
-
-Reaction.registerPackage({
-  label: "Address",
-  name: "reaction-address",
-  autoEnable: true,
-  functionsByType: {
-    registerPluginHandler: [registerPluginHandler]
-  },
-  graphQL: {
-    resolvers,
-    schemas
-  },
-  queries,
-  registry: [
-    {
-      label: "Address Validation",
-      provides: ["shopSettings"],
-      container: "dashboard",
-      template: "ShopAddressValidationSettings"
-    }
-  ]
-});
+Reaction.whenAppInstanceReady(register);
```

Comment:
I'm not a fan of this organization, as opposed to having the contents of that register file here, in this register file.

Reply:
This is necessary temporarily to be backwards compatible while finishing removing Meteor. Soon this file will go away and only the inner `register.js` will remain, but currently this file is what our Meteor tooling loads.

Reply:
I added a comment with this explanation at the top of all these files. Thanks for calling it out as confusing.